### ISSUE TITLE
Need netCDF4 in requirements.txt

### ISSUE THREAD

**Issue by mhidas** [ioos/compliance-checker#272]
The netCDF4 package isn't currently specified in the requirements.txt file. When I run:
```
$ pip install compliance-checker
...
Successfully installed Jinja2-2.8 MarkupSafe-0.23 OWSLib-0.11.2 cf-units-1.0.0 compliance-checker-2.2.0 isodate-0.5.4 lxml-3.6.0 pygeoif-0.6 pyproj-1.9.5.1 python-dateutil-2.5.3 pytz-2016.4 requests-2.10.0 six-1.10.0
```

But...
```
$ compliance-checker -h
Traceback (most recent call last):
  File "/home/mhidas/code/env/mycc/bin/compliance-checker", line 7, in <module>
    from cchecker import main
  File "/home/mhidas/code/env/mycc/bin/cchecker.py", line 6, in <module>
    from compliance_checker.runner import ComplianceChecker, CheckSuite
  File "/home/mhidas/code/env/mycc/local/lib/python2.7/site-packages/compliance_checker/runner.py", line 8, in <module>
    from compliance_checker.suite import CheckSuite
  File "/home/mhidas/code/env/mycc/local/lib/python2.7/site-packages/compliance_checker/suite.py", line 12, in <module>
    from netCDF4 import Dataset
ImportError: No module named netCDF4
```

I'm just not sure which version of netCDF4 should be specified as a minimum, if any.

**Comment by daf** [comment]
Sorry @mhidas!  When [ioos/wicken](github.com/ioos/wicken) was a requirement, it ahd the dependency on netCDF4, and we were using that at the time.  This is an obvious omission.

**Comment by daf** [comment]
Fixed in afd9a87bb4ff4649aac772004e19d5eec95ec6bc
